src/shared/betaviz.js

```diff
@@ -0,0 +1,403 @@
+import calculations from "./calculations";
+import * as d3 from "d3";
```

Comment:
It seems like the only thing keeping you from being able to selectively import only the stuff you need is that currently you access the cmap dynamically from d3. If you do need that, a solution would be to import `interpolateSpectral` and others you might use and group them on a colorsMap object.

Reply:
Looks like this suggestion was applied. Resolving